pillar.example

```diff
@@ -1,110 +1,119 @@
 dovecot:
-  lookup:
```

Comment:
`Lookup` should not be removed, it's required for the map.jinja to work, items should be underneath the lookup section to ensure it merges correctly.